extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -27,7 +27,10 @@ class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' )
 	 * @return string
 	 */
 	function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
-		Jetpack_Gutenberg::load_assets_as_required( 'tiled-gallery' );
+		Jetpack_Gutenberg::load_assets_as_required(
+			'tiled-gallery',
+			array( 'wp-polyfill' )
```

Comment:
👍

Reply:
do we care about the order?

Reply:
This is `public static function load_assets_as_required( $type, $script_dependencies = array() )`, it will find the `tiled-gallery` view script and enqueue it with the `wp-polyfill` dependency. The order is good here 👍